development/docker-compose.yml

```diff
@@ -145,6 +134,19 @@ services:
       INFRAHUB_DB_PORT: 7687
       INFRAHUB_DB_PROTOCOL: bolt
       INFRAHUB_STORAGE_DRIVER: local
+      # INFRAHUB_SERVER_COMMAND is defined while running with reload option as it requires to run server using `uvicorn`.
+      COMMAND: ${INFRAHUB_SERVER_COMMAND:- gunicorn --config backend/infrahub/serve/gunicorn_config.py -w ${WEB_CONCURRENCY:-4} --logger-class infrahub.serve.log.GunicornLogger infrahub.server:app --reload}
```

Comment:
I think the `--reload` here is a leftover from some previous test as we don't want that to be default only when the invoke command sets the `INFRAHUB_SERVER_COMMAND` environment variable.

Reply:
Yes good catch